Makefile

```diff
@@ -1,3 +1,22 @@
+ALL_SRC := $(shell find . -name '*.go' \
+								-not -path './vendor/*' \
+								-not -path '*/gen-go/*' \
+								-type f | sort)
+ALL_PKGS := $(shell go list $(sort $(dir $(ALL_SRC))))
+
+# TODO:[rghetia] eventually remove this as all pkgs needs test coverage
+ALL_TEST_SRC := $(shell find . -name '*_test.go' \
+								-not -path './vendor/*' \
+								-not -path '*/gen-go/*' \
```

Comment:
I'm confused as to why this and the ALL_SRC stuff is even needed. Go list filters out vendor by default (for a while now) and generated code shouldn't have any tests.

Reply:
> I'm confused as to why this and the ALL_SRC stuff is needed. `go list` filters out vendor by default (for a while now) and generated code shouldn't have any tests.

When I ran test coverage I got
```
can't load package: package github.com/open-telemetry/opentelemetry-go: build constraints exclude all Go files in /Users/rahulpa/rghetia/go/src/github.com/open-telemetry/opentelemetry-go
```
It mislead me to think that it is an error and the test will fail. So to avoid it I ended up adding ALL_TEST_SRC (modified from opencensus-go [Makefile](https://github.com/census-instrumentation/opencensus-go/blob/19a91518e33fbd42c94402bb5758d88f3c3738eb/Makefile#L2-L6))

There is no need to add ALL_TEST_SRC. I'll remove it.